src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java

```diff
@@ -174,11 +180,28 @@ public List<Installable<PluginDescriptor>> process(List<EntandoBundleComponentJo
                 .collect(Collectors.toList());
     }
 
-    @Override
-    public PluginDescriptor buildDescriptorFromComponentJob(EntandoBundleComponentJobEntity component) {
-        return new PluginDescriptor()
-                .setDescriptorMetadata(
-                        new DescriptorMetadata(null, null, null, null, component.getComponentId(), null, null));
+    /**
+     * Reads the plugin descriptor and in case adjust its registry address if not present in the plugin image url.
+     */
+    private static PluginDescriptor mkAdjustedPluginDescriptor(BundleReader bundleReader, String filename,
+            CraneCommand craneCommand) throws IOException {
+        var pluginDescriptor = bundleReader.readDescriptorFile(filename, PluginDescriptor.class);
+
+        String fqImageAddress = getFqImageAddress(pluginDescriptor, bundleReader);
+        pluginDescriptor.setDockerImage(DockerImage.fromString(fqImageAddress));
```

Comment:
I think we should log at debug level the calculated FullyQualifiedImage before use it